_src/pl/static/sass/_base.scss

```diff
@@ -23,7 +23,7 @@ html, body {
 
 .pl {
     background: palette(grayscale, x-light);
-    font-size: $font-scale-medium;
+    font-size: font-size(base);
     font-family: $font-stack-sans-serif;
     -webkit-font-smoothing: antialiased;
```

Comment:
Yeah, there were no utilities out of the box for this, but good question/point. I can easily add a simple edX mixin like the one here - http://maximilianhoffmann.com/posts/better-font-rendering-on-osx.

I'll do that! Thx!